django_loci/admin.py

```diff
@@ -39,6 +41,12 @@ class ObjectLocationInline(AbstractObjectLocationInline):
     model = ObjectLocation
     form = ObjectLocationForm
 
+_admin_site_get_urls = admin.site.get_urls
 
+def get_urls():
+    return _admin_site_get_urls() + [url(r'^geocode/$', admin.site.admin_view(GeoCodeView.as_view()))]
```

Comment:
why are you putting this hear instead of `base.admin`, also, why can't you override the `get_urls` as we do in the other openwisp-modules?

Reply:
I think it would be cleaner to have one URL for geocoding and one for reverse geocoding